app/javascript/components/enrollment/steps/CaseInformation.js

```diff
@@ -0,0 +1,512 @@
+              minDate={'2020-01-01'}
+              maxDate={moment().format('YYYY-MM-DD')}
+              onChange={this.handleSymptomOnsetChange}
+              placement="bottom"
+              isInvalid={!!this.state.errors['symptom_onset']}
+              isClearable={!!this.props.patient.user_defined_symptom_onset || !this.props.patient.symptom_onset}
+              customClass="form-control-lg"
+              ariaLabel="Symptom Onset Date Input"
+            />
+            <Form.Control.Feedback className="d-block" type="invalid">
+              {this.state.errors['symptom_onset']}
+            </Form.Control.Feedback>
+          </Form.Group>
+          <Form.Group as={Col} md={12} xs={24} controlId="first_positive_lab" className="mb-2">
+            <FirstPositiveLaboratory lab={this.state.current.first_positive_lab} onChange={this.handleLabChange} size="lg" displayedLabClass="mx-1 mb-2" />
+          </Form.Group>
+          <Form.Group as={Col} md={12} xs={24} controlId="case_status" className="mb-2">
+            <Form.Label className="input-label">CASE STATUS{schema?.fields?.case_status?._exclusive?.required && ' *'}</Form.Label>
+            <Form.Control
+              isInvalid={this.state.errors['case_status']}
+              as="select"
+              size="lg"
+              className="form-square"
+              aria-label="Case Status Select"
+              onChange={this.handleChange}
+              value={this.state.current.patient.case_status || ''}>
+              <option></option>
+              <option>Confirmed</option>
+              <option>Probable</option>
+            </Form.Control>
+            <Form.Control.Feedback className="d-block" type="invalid">
+              {this.state.errors['case_status']}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} md="24" controlId="exposure_notes" className="mb-2">
+            <Form.Label className="input-label ml-1">NOTES{schema?.fields?.exposure_notes?._exclusive?.required && ' *'}</Form.Label>
+            <Form.Control
+              isInvalid={this.state.errors['exposure_notes']}
+              as="textarea"
+              rows="4"
+              size="lg"
+              className="form-square"
+              placeholder="enter additional information about case"
+              maxLength="2000"
+              value={this.state.current.patient.exposure_notes || ''}
+              onChange={this.handleChange}
+            />
+            <div className="character-limit-text">{2000 - (this.state.current.patient.exposure_notes || '').length} characters remaining</div>
+            <Form.Control.Feedback className="d-block" type="invalid">
+              {this.state.errors['exposure_notes']}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+      </React.Fragment>
+    );
+  };
+
+  renderPublicHealthManagementFields = () => {
```

Comment:
can this section be a separate component used by exposure and isolation or was there a problem with that?

Reply:
This was how @mdube7 originally did it at my suggestion but ultimately is what led to issues with the validation (the issue we talked about the other day).  Nothing I tried passing back and forth was able to resolve that so that's why I ultimately reverted to this.  I don't love it but otherwise the validation is not going to work which IMO is a much bigger issue

Reply:
ok that makes sense, should we just add a comment letting future devs know to make updates to both components if they make any changes to this?

Reply:
yup added here: https://github.com/SaraAlert/SaraAlert/pull/1027/commits/6729c540b5c62c776a59e35ef68d0f55d2ec04e7

Reply:
updated this to be it's own component now complete with front end tests, please test thoroughly!